city_scrapers/spiders/chi_ohare_noise.py

```diff
@@ -0,0 +1,212 @@
+import re
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import (
+    CANCELLED,
+    COMMISSION,
+    COMMITTEE,
+    NOT_CLASSIFIED,
+    PASSED,
+    TENTATIVE,
+)
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy import Request
+
+
+class ChiOhareNoiseSpider(CityScrapersSpider):
+    name = "chi_ohare_noise"
+    agency = "Chicago O'Hare Noise Compatibility Commission"
+    timezone = "America/Chicago"
+
+    class ChiOhareNoiseSubSpider1:
+        def parse(self, response):
+            """
+            `parse` should always `yield` Meeting items.
+
+            Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+            needs.
+            """
+            for item in response.css(".ev_td_li"):
+                surl = self._parse_url(item)
+                yield Request(url=response.urljoin(surl), callback=self._parse_details)
+
+            next_page = response.xpath("//div[@class='previousmonth']/a/@href").get()
+            if next_page is not None:
+                yield response.follow(response.urljoin(next_page), callback=self.parse)
+
+        def _parse_details(self, response):
+            stime = self._parse_start(response)
+            meeting = Meeting(
+                title=self._parse_title(response).replace("CANCELLED ", "").strip("- "),
+                description=self._parse_description(response),
+                start=stime,
+                end=stime + timedelta(hours=1),
+                all_day=False,
+                location=self._parse_location(response),
+                links=self._parse_links(response),
+                source=response.url,
+            )
+
+            meeting = self._get_status(meeting)
+            meeting = self._parse_classification(meeting)
+            return meeting
+
+        def _parse_title(self, response):
+            """Parse or generate meeting title."""
+            return response.xpath(
+                "//div[@class='jev_evdt_header']/div/h2/text()"
+            ).extract()[0]
+
+        def _parse_url(self, response):
+            """Parse or generate meeting title."""
+            return [i.strip() for i in response.xpath("p/a/@href").extract()][0]
+
+        def _parse_description(self, response):
+            """Parse or generate meeting description."""
+            return response.xpath("//div[@class='jev_evdt_desc']/p/text()").get() or ""
+
+        def _parse_classification(self, meeting):
+            """Parse or generate classification from allowed options."""
+            if "committee" in meeting["title"].lower():
+                meeting["classification"] = COMMITTEE
+            elif "commission" in meeting["title"].lower():
+                meeting["classification"] = COMMISSION
+            else:
+                meeting["classification"] = NOT_CLASSIFIED
```

Comment:
Since the overall organization is a commission it's safe to default to `COMMISSION` instead of `NOT_CLASSIFIED`